### PR TITLE
Feature/frequency native set

### PR DESCRIPTION
**Pull request by anerhan**

### CODE REVIEW

File: spec/factories/account.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
 FactoryBot.define do
   factory :source_account do
     uid { SecureRandom.hex(30) }
-    current_balance { 0.0 }
-    available_balance { 0.0 }
+    balance { 0.0 }
```
Comment (agm1988) [inline spec/factories/account.rb]: braces not needed unless it's callable code

File: spec/models/invest_set_spec.rb [hunk 2]
```diff
@@ -19,15 +19,56 @@
     it { is_expected.to have_db_index(:invest_account_id) }
   end
 
+  context 'constants' do
+    describe 'FREQUENCIES' do
+      specify 'exist' do
+        expect(InvestSet::FREQUENCIES).to eq(%w[once daily weekly monthly])
```
Comment (agm1988) [inline spec/models/invest_set_spec.rb]: order matters? maybe `match_array`?

File: spec/models/invest_set_spec.rb [hunk 3]
```diff
@@ -19,15 +19,56 @@
...
   end
+
+  let(:user) { create :user, :with_invest_sets }
+  let(:invest_set) { user.current_invest_set }
+  let(:new_invest_set) { InvestSet.new }
+
+  context 'Instance methods' do
+    describe '#ready?' do
+      specify 'true if it has ready source & Invest accounts, and if it is not new record' do
+        expect(invest_set.ready?).to be_truthy
```
Comment (agm1988) [inline spec/models/invest_set_spec.rb]: here you return boolean, better to use `be true`, to be more explicit.

File: app/services/plaid_service.rb [hunk 4]
```diff
@@ -23,6 +23,8 @@ def accounts
       attrs[:dwolla_token] = self.class.fetch_dwolla_processor_token(access_token, attrs[:uid])
       PlaidAccount.new(attrs)
     end
+  rescue StandardError => error
```
[the remark this answers is not in the document]
Comment (anerhan) [reply app/services/plaid_service.rb]: In Plaid gem all exceptions wrapped to same structure, and they are has standart accessors to message, code ... Using StandartError I can handle all exceptions catched by this method and return defined structured json in response from controller

File: spec/factories/account.rb (hunk 1 above)
Comment (anerhan) [reply spec/factories/account.rb]: callable values is in deprecation
```
DEPRECATION WARNING: Static attributes will be removed in FactoryBot 5.0. Please use dynamic
attributes instead by wrapping the attribute value in a block:
```

File: spec/models/invest_set_spec.rb (hunk 2 above)
Comment (anerhan) [reply spec/models/invest_set_spec.rb]: agree

File: spec/models/invest_set_spec.rb (hunk 3 above)
Comment (anerhan) [reply spec/models/invest_set_spec.rb]: Agree
```
expect(actual).to be_truthy    # passes if actual is truthy (not nil or false)
expect(actual).to be true      # passes if actual == true
```

File: app/helpers/invest_sets_helper.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+module InvestSetsHelper
+  def frequency_checked?(value)
+    return unless @invest_set
```
[the remark this answers is not in the document]
Comment (anerhan) [reply app/helpers/invest_sets_helper.rb]: Instance variable @invest_set defined in before_action in DashboardController

[the remark this answers is not in the document]
Comment (agm1988) [reply app/helpers/invest_sets_helper.rb]: ok

File: app/services/plaid_service.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (agm1988) [reply app/services/plaid_service.rb]: :+1:

File: spec/factories/account.rb (hunk 1 above)
Comment (agm1988) [reply spec/factories/account.rb]: ok

**Comment by agm1988** [comment]
:+1:

**Pull request merged**